### ISSUE TITLE
Lists missing label on select element when aria-label is present.

### ISSUE THREAD

**Issue by leegee** [GoogleChrome/lighthouse#9294]
#### Environment Information
* Affected Channels: Chrome extension
* Lighthouse version: 4.3.1
* Node.js version: N/A
* Operating System: OSX 10.14.5

The reported fault is that a select element does not have a label — LH links to a page that states adding an `aria-label` attribute will resolve the issue. As you can see in the below screenshot, an `aria-label` is present — unless I've missed something?

![image](https://user-images.githubusercontent.com/512816/60266068-2a850380-98df-11e9-876a-7a64c71c22bb.png)

**Comment by patrickhulce** [comment]
Thanks for filing @leegee. 

This potential issue would be with axe would you mind filing in [their repo](https://github.com/dequelabs/axe-core/issues/new)? They would also likely need a repro URL in order to help.

**Comment by Rombecchi** [comment]
I have some "select" and "text input" elements in a form  with explicit tags "label", called by ID,  with CSS "display:none".
Lighthouse reported me that there were no associated labels, so I put the "aria-label" attr to each element, but without any luck....

**Comment by brendankenny** [comment]
@Rombecchi that could be the same issue @leegee reported in https://github.com/dequelabs/axe-core/issues/1663 or something slightly different. If it is the same, it looks like it would help a lot if you could provide an example or url that reproduces the problem for them. If it's not, @patrickhulce's advice still applies:

> This potential issue would be with axe would you mind filing in [their repo](https://github.com/dequelabs/axe-core/issues/new)? They would also likely need a repro URL in order to help.

**Comment by Rombecchi** [comment]
@brendankenny here you are the link for test: 
[https://demo.agestaweb.it/web/default.asp?fr=1&force=1&a=58cbe0e13c5db3aef0cf901f68f291db&preview=1&e=8586](https://demo.agestaweb.it/web/default.asp?fr=1&force=1&a=58cbe0e13c5db3aef0cf901f68f291db&preview=1&e=8586)

Tested with:
- Desktop mode
- Performance, Best practices, Accessibility, SEO
- No throttling
- Clear storage: ON


Failing Elements
- select#tipo_contratto.form-control
- select#cod_comune.form-control
- select#prezzo_max.form-control.fascia
- input#riferimento.input_text.form-control


Thanks

**Comment by brendankenny** [comment]
@Rombecchi, sorry, I meant report in the axe repo! That's where the bug fix will actually happen, and Lighthouse will pick it up once it's released.

**Comment by patrickhulce** [comment]
Without a repro we can't do much here, if anyone can find this please file in the axe-core repo :)
